proto/tendermint/consensus/types.proto

```diff
@@ -79,14 +86,15 @@ message VoteSetBits {
 
 message Message {
   oneof sum {
-    NewRoundStep  new_round_step  = 1;
-    NewValidBlock new_valid_block = 2;
-    Proposal      proposal        = 3;
-    ProposalPOL   proposal_pol    = 4;
-    BlockPart     block_part      = 5;
-    Vote          vote            = 6;
-    HasVote       has_vote        = 7;
-    VoteSetMaj23  vote_set_maj23  = 8;
-    VoteSetBits   vote_set_bits   = 9;
+    NewRoundStep               new_round_step          = 1;
+    NewValidBlock              new_valid_block         = 2;
+    Proposal                   proposal                = 3;
+    ProposalPOL                proposal_pol            = 4;
+    BlockPart                  block_part              = 5;
+    Vote                       vote                    = 6;
+    HasVote                    has_vote                = 7;
+    VoteSetMaj23               vote_set_maj23          = 8;
+    VoteSetBits                vote_set_bits           = 9;
+    HasProposalBlockPart       has_proposal_block_part = 10;
```

Comment:
Do we have a policy on altering the protocol on the backport branches?
I guess this works if all nodes in the network run software based on the 0.38.x-experimental branch _starting from a certain revision_, but should there be more discipline to how we maintain these branches, so their consumers can get some measure of stability?

Reply:
Or maybe I'm overthinking this and experimental is really what it says in the branch name, use it at your own risk.

Reply:
My understanding is that even if some nodes run with the non-experimental branch, they would error out and discard the messages they cannot desserialize. Is this wrong?

There is an interesting point here with respect to compatibility of improved base and experimental branchs. If the base is modified, it should not redefine experimental fields as it would require a coordinated update to also update to an updated experimental branch.

Also, FYI, eventually we may bring this feature in the base branch.